client/src/components/DashboardBarMenu/DashboardBarMenu.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, MouseEvent } from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuIcon from '@material-ui/icons/Menu';
+import { useAuth } from '../../context/useAuthContext';
+
+const DashboardBarMenu = (): JSX.Element => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const { logout } = useAuth();
+
+  const handleClick = (event: MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const handleLogout = () => {
+    handleClose();
+    logout();
+  };
+
+  return (
+    <div>
+      <IconButton
+        aria-label="show dashboard bar menu"
+        aria-controls="dashboard bar menu"
+        aria-haspopup="true"
+        onClick={handleClick}
+      >
+        <MenuIcon />
+      </IconButton>
```

Comment:
This is actually in the wrong place, I know the mock-up isn't clear but the dropdown menu is where we click on the profile picture it will show these options. The hamburger menu icon is clicked to show all the users boards. 

Let's think of a better icon to use that's more descriptive.